roles/cifmw_cephadm/tasks/logs.yml

```
@@ -0,0 +1,32 @@
+- name: Get ceph_cli
+  ansible.builtin.include_tasks: ceph_cli.yml
+
+- name: Get the ceph orchestrator status
+  ansible.builtin.command: "{{ cifmw_cephadm_ceph_cli }} orch status --format json"
+  register: ceph_orch_status
+  become: true
+
+- name: Fail if ceph orchestrator is not available
+  ansible.builtin.fail:
+    msg: "'ceph orch status' returned {{ ceph_orch_status.stdout | from_json }}"
```

mmh, probably we should improve the renderinof the output here. I was thinking something about:

```suggestion
    msg: |
      'ceph orch status' returned 
      {{ ceph_orch_status.stdout | from_json }}
```

---

Done